mozilla_bitbar_devicepool/configuration.py

```diff
@@ -5,12 +5,14 @@
 from __future__ import absolute_import
 
 import os
```

Comment:
```
import threading
```

Reply:
Why? I can't find any usage of the module in configuration.py.